pynecone/state.py

```diff
@@ -180,8 +183,18 @@ def __init_subclass__(cls, **kwargs):
         }
         for name, fn in events.items():
             event_handler = EventHandler(fn=fn)
+            cls.event_handlers[name] = event_handler
             setattr(cls, name, event_handler)
 
+    @classmethod
+    def convert_handlers_to_fns(cls):
+        """Convert the event handlers to functions.
+
+        This is done so that the event handlers can be pickled.
```

Comment:
I think we should say this is done so that we can call the state functions as normal functions during runtime